ubg/src/common/SettingsModal.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import FormControl from '@material-ui/core/FormControl';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import {makeStyles} from '@material-ui/core/styles';
+import {useState, useEffect} from 'react';
+import PropTypes from 'prop-types';
+
+const useStyles = makeStyles((theme) => ({
+  fonts: {
+    fontWeight: 'bold',
+    color: theme.palette.primary.main,
+  },
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 120,
+  },
+}));
+
+/**
+ * @param {object} usersData User collection data
+ * @param {object} usersCollection Users collection
+ * @param {func} startGame Updates the game to start
+ * @return {ReactElement} Mafia modal element
+ */
+export default function Settings({usersData, usersCollection, startGame}) {
```

Comment:
I think normally we want to call the component the same as the file, and I don't know what you think about moving it to the mafia folder since this modal is just for the mafia game.

Reply:
So move it into the mafia folder?